recipes/cpuinfo/all/conanfile.py

```diff
@@ -72,6 +72,8 @@ def _configure_cmake(self):
         self._cmake.definitions["CLOG_RUNTIME_TYPE"] = "default"
         self._cmake.definitions["CLOG_BUILD_TESTS"] = False
         self._cmake.definitions["CMAKE_POSITION_INDEPENDENT_CODE"] = self.options.get_safe("fPIC", True)
+        if self.settings.arch == "armv8":
+            self._cmake.definitions["CMAKE_SYSTEM_PROCESSOR"] = "armv8"
```

Comment:
Passing `CMAKE_SYSTEM_PROCESSOR` as argument without a `CMAKE_TOOLCHAIN_FILE`, does this set `CMAKE_CROSSCOMPILING`?

Reply:
Thanks for the reply.

I was confusing `CMAKE_SYSTEM_NAME` and `CMAKE_SYSTEM_PROCESSOR`.
Both are supposed to be set by cmake toolchains.
Accidentally enabling `CMAKE_CROSSCOMPILING` might modify the build logic somewhere unwanted.

But we're good here. Thanks!